packages/api/src/guards/roles.Guard.ts

```diff
@@ -0,0 +1,28 @@
+import { CanActivate, ExecutionContext, Injectable, SetMetadata } from '@nestjs/common';
```

Comment:
💭 Name of this file should be `roles.guard.ts` as we write all backend file names in kebab-case